### ISSUE TITLE
Implement project scope support

### ISSUE THREAD

**Issue by maxfrigge** [tomkis1/js-autoimport#5]
I don't get any of modules in code completion.. is there anything I should comply with to get this working?

**Comment by tomkis** [comment]
No, it should be working. Could you please provide further information? Console log would be much appreciated.

**Comment by maxfrigge** [comment]
I had another go at it and it seems to work now - quite well to be honest. 

Just 2 things I have noticed:
- The automatic ordering of imports can be annoying when manually importing
- I tend to have multiple projects in my workspace which miss leads the importer. Is there a setting to define scope?

**Comment by tomkis** [comment]
The former is known bug, it should not touch the imports while they are being manually edited. The latter is unfortunately not supported at the moment.
